Resources/Prototypes/_Goobstation/Entities/Objects/Tools/jetpacks.yml

```
@@ -33,3 +33,41 @@
+      outputPressure: 42.6
+      air:
+        volume: 1
+
+# Filled improvised - GoobStation
+- type: entity
+  id: JetpackImprovisedFilled
+  parent: JetpackImprovised
+  name: improvised jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)